docs/configuration.md

```diff
@@ -31,6 +53,24 @@ extensions:
       namespace: default
 ```
 
+### Configuration Validation
+
+k0s command-line interface has the ability to validate config syntax:
+
+```
+$ k0s validate config --config path/to/config/file
+```
+
+`validate config` sub-command can validate the following:
+
+1. YAML formatting
+2. [SANs addresses](#specapi-1)
+3. [Network providers](#specnetwork-1)
+4. [Worker profiles](#specworkerprofiles) 
+
+
+## Configuration Keys
+
 ### `spec.api`
 
 - `address`: The local address to bind API on. Also used as one of the addresses pushed on the k0s create service certificate on the API. Defaults to first non-local address found on the node.
```

Comment:
we still have two sections describing spec.api, spec.network and helm extensions. one under the "configuration keys" and also under "full configuration".
It's not clear why we have both, so I would merge the configuration keys and the full configuration sections (and rename "full configuration" to "configuration keys")

Reply:
I think it would be better to remove those examples and just add configurations specs instead of this